edit_rule_ui.gs

```diff
@@ -0,0 +1,187 @@
+/**
```

Comment:
It seems that generate_report_ui and this has too many code redundant. Refactor the code to extract the common functionality

Reply:
Card Service has limitations
I tried some things but wasn't successful